src/main/cpp/states/indexer/IndexerStateMgr.cpp

```diff
@@ -120,6 +122,10 @@ void IndexerStateMgr::CheckForStateTransition()
                 targetState = INDEXER_STATE::INDEX_RIGHT;
             }
         }
+        else if (controller->IsButtonPressed(TeleopControl::FUNCTION_IDENTIFIER::MANUAL_INDEX))
+        {

```

Comment:
Probably should be first in the if/else block; otherwise if the sensor is broken and doesn't detect a ball, we won't be able to get balls from the left.